tests/components/signal_messenger/test_notify.py

```diff
@@ -91,6 +92,27 @@ def send_message_with_attachment(signal_notification_service, deprecated=False):
         signal_notification_service.send_message(MESSAGE, **{"data": data})
 
 
+def test_send_message_with_attachment_as_url(
+    signal_notification_service, signal_requests_mock, caplog
+):
+    """Test send message with attachment."""
+    with caplog.at_level(
+        logging.DEBUG, logger="homeassistant.components.signal_messenger.notify"
+    ):
+        send_message_with_attachment_as_url(signal_notification_service)
+
+    assert "Sending signal message" in caplog.text
+    assert signal_requests_mock.called
+    assert signal_requests_mock.call_count == 3
+    assert_sending_requests(signal_requests_mock, 1)
+
```

Comment:
This year really aight to check the content of the data sent.

Reply:
Added content check